CHANGELOG.md

```diff
@@ -6,8 +6,11 @@ title: Changelog
 
 ## Unreleased (????-??-??)
 
-* colors: use variables to specify colors and swap old colors for new contrasted ones.
+* colors: use variables to specify colors and swap old colors for new contrasted ones. Add ADR about colors and Design Tokens.
```

Comment:
Not sure this can be in the same commit and PR but since we're not going to create a commit and PR for this only, I figured I might as well add it here :roll_eyes: